infrastructure/cdn-in-a-box/traffic_ops/config.sh

```diff
@@ -100,108 +92,103 @@ cat <<-EOF >/opt/traffic_ops/app/conf/cdn.conf
         "log_location_info": "$TO_LOG_INFO",
         "log_location_debug": "$TO_LOG_DEBUG",
         "log_location_event": "$TO_LOG_EVENT",
-        "max_db_connections": 20,
         "db_conn_max_lifetime_seconds": ${DEBUGGING_TIMEOUT:-60},
-        "db_query_timeout_seconds": ${DEBUGGING_TIMEOUT:-20},
-        "backend_max_connections": {
-            "mojolicious": 4
-        },
-        "whitelisted_oauth_urls": [],
-        "oauth_client_secret": "",
-        "routing_blacklist": {
-            "ignore_unknown_routes": false,
-            "disabled_routes": []
-        },
-        "supported_ds_metrics": [ "kbps", "tps_total", "tps_2xx", "tps_3xx", "tps_4xx", "tps_5xx" ]
-    },
-    "cors" : {
-        "access_control_allow_origin" : "*"
+        "db_query_timeout_seconds": ${DEBUGGING_TIMEOUT:-20}
     },
     "to" : {
-        "base_url" : "https://$TO_FQDN",
-        "email_from" : "no-reply@$INFRA_SUBDOMAIN.$TLD_DOMAIN",
-        "no_account_found_msg" : "A Traffic Ops user account is required for access. Please contact your Traffic Ops user administrator."
+        "email_from" : "no-reply@$INFRA_SUBDOMAIN.$TLD_DOMAIN"
     },
     "portal" : {
         "base_url" : "https://$TP_HOST.$INFRA_SUBDOMAIN.$TLD_DOMAIN/#!/",
-        "email_from" : "no-reply@$INFRA_SUBDOMAIN.$TLD_DOMAIN",
-        "pass_reset_path" : "user",
-        "user_register_path" : "user"
-    },
-    "secrets" : [
-        "$TO_SECRET"
-    ],
-    "geniso" : {
-        "iso_root_path" : "/opt/traffic_ops/app/public"
+        "email_from" : "no-reply@$INFRA_SUBDOMAIN.$TLD_DOMAIN"
     },
-    "inactivity_timeout" : 60,
     "smtp" : {
         "enabled" : true,
-        "user" : "",
-        "password" : "",
         "address" : "${SMTP_FQDN}:${SMTP_PORT}"
     },
     "InfluxEnabled": true,
     "influxdb_conf_path": "/opt/traffic_ops/app/conf/production/influx.conf",
     "lets_encrypt" : {
-        "user_email" : "",
-        "send_expiration_email": false,
-        "convert_self_signed": false,
-        "renew_days_before_expiration": 30,
         "environment": "staging"
-    },
-    "acme_renewal": {
-        "summary_email": "",
-        "renew_days_before_expiration": 30
-    },
-    "acme_accounts": [
-        {
-            "acme_provider" : "",
-            "user_email" : "",
-            "acme_url" : "",
-            "kid" : "",
-            "hmac_encoded" : ""
-        }
-    ]
```

Comment:
I don't think `postinstall` will add LE/ACME things.

Reply:
Re-added LE/ACME things in baf2134eda

Reply:
Removed it again. The default `cdn.conf` includes these exact LE/ACME settings.